nixos/modules/services/audio/liquidsoap.nix

```diff
@@ -39,9 +39,9 @@ in
       default = {};
 
       example = {
-        myStream1 = literalExample "\"/etc/liquidsoap/myStream1.liq\"";
-        myStream2 = literalExample "./myStream2.liq";
-        myStream3 = literalExample "\"out(playlist(\\\"/srv/music/\\\"))\"";
+        myStream1 = "/etc/liquidsoap/myStream1.liq";
+        myStream2 = literalExpression "./myStream2.liq";
+        myStream3 = "out(playlist(\"/srv/music/\"))";
```

Comment:
Yes, because of https://github.com/NixOS/nixpkgs/blob/7767b5cc6ab70829d3f92eca4e7bd211d6c1aaa3/nixos/lib/make-options-doc/options-to-docbook.xsl#L176-L178
not sure it should be supported though. (I didn't add this)